encoders/image/ImageTorchEncoder/__init__.py

```diff
@@ -74,6 +76,14 @@ def _get_pooling(self, feature_map: 'np.ndarray') -> 'np.ndarray':
     @batching
     @as_ndarray
     def encode(self, data: 'np.ndarray', *args, **kwargs) -> 'np.ndarray':
+        """
+        Encode input data into `np.ndarray`.
+        :param data: Image to be encoded, expected a `np.ndarray` of BatchSize x (Channel x Height x Width).
```

Comment:
```suggestion

        :param data: Image to be encoded, expected a `np.ndarray` of BatchSize x (Channel x Height x Width).
```

Reply:
Image/Images to be encoded, expected a `np.ndarray` of shape  BatchSize x (Channel x Height x Width)